src/styles/vars.css

```diff
@@ -4,11 +4,13 @@
   --midnight-blue: #212F3D;
   --cloud: #ECF0F1;
   --cloud-dark: #d3d8da;
+  --cloud-darker: #b5b9bb;
 
   /* Colours used by other modules */
   --accent-1-colour: var(--green-sea);
   --light-base-colour: var(--cloud);
   --light-base-colour-dark: var(--cloud-dark);
+  --light-base-colour-darker: var(--cloud-darker);
   --dark-base-colour: var(--midnight-blue);
```

Comment:
@maxkfranz is this what you had in mind for semantic css var names?  I find it more difficult to remember ```accent-1-colour``` vs ```green-sea```.